### PR TITLE
Some vaults

### PR DESCRIPTION
**Pull request by alexjurkiewicz**

### CODE REVIEW

**Comment by alexjurkiewicz** [comment]
should have fixed the final remaining syntax bug in this

**Comment by alexjurkiewicz** [comment]
Updated. For e6d32d0 there could potentially be more loot (I significantly reduced the amount).

File: crawl-ref/source/dat/des/altar/overflow.des [hunk 1]
```diff
@@ -3845,3 +3845,51 @@ MAP
...
+MAP
+.....
+..B..
+.BGB.
+..B..
+.....
+ENDMAP
+
+NAME:   chequers_temple_overflow_statue_of_worship_2
+TAGS:   temple_overflow_generic_2
```
Comment (yrmvgh) [inline crawl-ref/source/dat/des/altar/overflow.des]: orient float on this one?

**Comment by alexjurkiewicz** [comment]
ready for merge, I think

**Pull request closed without merging**